Make test_force_quant_dequant_fusion use GraphBuilder.

eigen-k opened the pull request:
Summary: Use GraphBuilder to create the model for unit testing.

Differential Revision: D74841541

mcremon-meta commented on backends/cadence/aot/tests/test_fusion_ops_passes.py:
```diff
@@ -248,32 +248,28 @@ def forward(self, x):
+            op=exir_ops.edge.quantized_decomposed.dequantize_per_tensor.default,
+            args=(permute, 4.5, 6, 0, 127, torch.int8),
+        )
+        builder.output(dequant)
```
Nit: use a variable for readability
```suggestion
        builder.output(dequant)
        graph_module = builder.get_graph_module()
```

mcremon-meta commented on backends/cadence/aot/tests/test_fusion_ops_passes.py:
```diff
@@ -248,32 +248,28 @@ def forward(self, x):
         graph_module = FuseQuantDequantToRequantizePass(
             force_quant_dequant_fusion=True
-        )(graph_module).graph_module
+        )(builder.get_graph_module()).graph_module
```
```suggestion
        )(graph_module).graph_module
```

eigen-k committed:
Make test_force_quant_dequant_fusion use GraphBuilder. (#10926)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/10926

Use GraphBuilder to create the model for unit testing.

Reviewed By: mcremon-meta, zonglinpeng

Differential Revision: D74841541
